spec/models/manageiq/providers/openstack/cloud_manager_spec.rb

```diff
@@ -225,65 +225,77 @@ def lastlog
   end
 
   context "validation" do
-    before :each do
-      @ems = FactoryBot.create(:ems_openstack_with_authentication)
-      require 'manageiq/providers/openstack/legacy/openstack_event_monitor'
-    end
+    let!(:ems) { FactoryBot.create(:ems_openstack_with_authentication) }
```

Comment:
Did some refactoring here to make it cleaner to override the existing ems, also the event_monitor_available? tests were mixed in to the hostname_uniqueness_valid? tests in the same context.

Refactoring is in a separate commit.